### ISSUE TITLE
PIP WONT TURN OFF

### ISSUE THREAD

**Issue by JackBurnell** [legendsayantan/ShizuTools#17]
PIP will not toggle off. Obtrusive PIP of my home screen stuck on. Tried reboot. MISTAKE!!!  Now something in here prevents me from starting wireless debugging, so I cant even start Shizuku. "An application is preventing authorization" message when I try to enable WDB. Uninstalled Tools. No help. My tablet is basically rendered useless for what I use it for without the ability to run Shizuku. How do I turn off that PIP. Probably should include a warning that it makes EVERYTHING PIP, and that you have no control of which apps it applies to.

**Comment by Sloom128** [comment]
Even me help

**Comment by Sloom128** [comment]
Any solutions?

**Comment by legendsayantan** [comment]
Disabling the pip through adb -

`adb shell settings put global overlay_display_devices null`

Or, if there is a "simulate secondary display" menu in your developer options, you can also set that to `none` to disable pip.

**Comment by JackBurnell** [comment]
Thank you. I already "fixed" it. But I discovered the problem when I
decided to be patient and look into it. The panel to toggle it pops up from
the bottom. On that tablet it never appeared. After I saw that on my other
phone I wrote a little script to change the screen density on the tablet.
Boom. Sorry if I was a bit impatient or sounded shitty. I.started
programming in the late 70s with Z-70 ASM and went from there, like we all
do, to many other languages and OSes. But Ive been out of pocket for the
last 25 years and everything relevant is something I dont know.  Javascript
as a full object oriented language suitable to write full apps with??? wha?
It was a data delivery boy for HTML last I saw it. And I have no computer
now, just a phone and fkn Android. My choices to get back in were Termux
and Python or Droidscript. Droidscript is at least a reasonable faccimile
of Javascript but Ive been SO frustrated at the crazy permissions in
Android and the complexity and range of object classes that Im pretty well
on edge. And Im old now and cant learn as fast as I used to. My kingdom
(which consists of some clothes and few worthless memoribilia from back
when I alive) for a PC I can run a Linux/Windows on. And enough RAM and
resources to run AndroidStudio, Delphi and some real Javascript development
on. Tiny little screens and keys, big hands, bad eyes and too many rules
"for my safety" are making me pretty intolerant. So sorry for the
impertinent post. I love Shizuku and anyone thats helping to humble the
beast that is Google. I owned a computer repair shop and store in the late
80s abd 90s. I told my customers like the day it came out "Thats a tool for
"them" to gather intel on us with". Even I didnt expect it to be what it is
now. Im blown away at how bad it is now. And Im carrying one in my pocket
as neutered and firewalled as much as possible. The world has gone insane.
And I think I might be too.
Thanks for everything.
